cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -173,7 +173,7 @@ public void execute() {
             //// check if output flag is set
             if (null != this.output) {
                 CommandUtil.printError(this.errStream,
-                        "'-o' and '--output' flag is only supported for building a single ballerina file.",
+                        "The '-o' and '--output' flag is only supported for building a single Ballerina file.",
```

Comment:
```suggestion
                        "The '-o' or '--output' flag is only supported when building a single Ballerina file.",
```

Reply:
Should be "and" since we need to refer to both flags. Changed to "The '-o' and '--output' flags are only supported for building a single Ballerina file.".

Reply:
The user can specify only one of the two. Either `-o` or `--output`

Reply:
This sentence refers to both flags. As in, both flags are supported only when building a single file. Therefore, changed as follows:

"he '-o' and '--output' flags are supported only when building a single Ballerina file."